tuna/rocmlir/rocmlir_tables.py

```diff
@@ -188,22 +189,83 @@ def __repr__(self) -> str:
 
   def config_string(self):
     """Return config as a flag/value string suitable for tuningRunner.py."""
-    string = "conv "  # +++pf:  of course generalise for gemm
-    for field, value in self.to_dict().items():
-      flag = self.options[field]
-      if flag:
+    string = "conv"
+    if self.data_type == 'f16':
+      string += 'fp16'
+    elif self.data_type == 'i8':
+      string += 'int8'
+    elif self.data_type == 'bf16':
+      string += 'bfp16'
+    string += " "
+
+    # In options order for canonicalisation, kind of.
+    for field, flag in self.options.items():
+      value = getattr(self, field, None)
+      if value is not None and flag is not None:
         string += f"{flag} {value} "
-    string += "-m conv"
+    string += "-t 1"  # Dummy "enable timing" option.
     return string
 
+  def parse_line(self, line):
```

Comment:
Moved from import_configs.py and into a class because gemm has one too, and it's different.